src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -174,6 +193,47 @@ private static async Task HandleSingleTypeAsync(CompletionContext context, Seman
                         filterText: memberDisplayName));
                 }
             }
+            else if (enclosingNamedType is not null)
```

Comment:
Does it work with top level statement?

Reply:
➡️ Yes, at least for the case of typing this:

```
int.Parse(empty)
```